### PR TITLE
Add sethmccombs as developer on sysdig-secure-plugin 

### PR DESCRIPTION
**Pull request by sethmccombs**
# Description

I am an engineer at Sysdig and would like to assist in the development of the sysdig-secure' plugin - https://github.com/jenkinsci/sysdig-secure-plugin
CC @markyjackson-taulia 

# Submitter checklist for changing permissions

### Always

- [X] Add link to plugin/component Git repository in description above

### CODE REVIEW

**Comment by halkeye** [comment]
this PR is failing

[2020-03-04T22:37:52.023Z] Exception in thread "main" java.lang.IllegalStateException: User name not known to Artifactory: sethmccombs

Do you have an account on https://accounts.jenkins.io ?

**Comment by markyjackson-taulia** [comment]
I am a +1 for this once the account is created and the job passes

File: permissions/plugin-sysdig-secure.yml [hunk 1]
```diff
@@ -7,3 +7,4 @@ developers:
 - "alfredlandrum"
 - "sysdig"
 - "jequals5"
+- "sethmccombs"
```
Comment (slide) [inline permissions/plugin-sysdig-secure.yml]: Looks like this is not the correct username for the Jenkins infra. Please verify this is the username you use to login to [Artifactory](https://repo.jenkins-ci.org/webapp/#/login).

**Comment by batmat** [comment]
Please resubmit without blanking out most of the PR template. A critical part exactly about this was removed :disappointed:.

PR template has been created for a reason: save time for you, and more especially from contributors reviewing these PRs on their free-time.

**Pull request closed without merging**